.github/workflows/compatability-tests.yaml

```diff
@@ -0,0 +1,300 @@
+name: Compatability Tests
```

Comment:
Can you please edit both release checklists (lakefs and hadoopfs) to include a step to update the matrixes in this test?

Reply:
I will as soon as this is merged!